tests/common/devices/sonic.py

```diff
@@ -1593,7 +1593,7 @@ def active_ip_interfaces(self, ip_ifs, ns_arg=""):
                 # Ping for some time to get ARP Re-learnt.
                 # We might have to tune it further if needed.
                 if (v["admin"] == "up" and v["oper_state"] == "up" and
-                   self.ping_v4(v["peer_ipv4"], count=10, ns_arg=ns_arg)):
+                   self.ping_v4(v["peer_ipv4"], count=3, ns_arg=ns_arg)):
```

Comment:
i needed to add to count fix some timing issue with qos. not sure if making 3 will break that. can we keep same. any particular reason for this change ?

Reply:
1 sec is actually good enough. The default timeout of ping is 4-5 secs. That's good enough to learn ARPs. With 10 count it adds more than 10 secs for each interface, increases the test running time without adding any benefit. I haven't seen any issues even with 1 sec.